spec/scrabble_spec.rb

```diff
@@ -0,0 +1,69 @@
+require './lib/scrabble'
+
+describe Scrabble::Scrabble do
+
+  describe "This method returns the score of a given word" do
```

Comment:
Hi @Elffers! Thanks for your feedback. I think I understand this better now after having written a lot more tests. Do you have any additional feedback/suggestions on how I've structured my tests and going about writing them?